.github/workflows/ci.yml

```diff
@@ -61,9 +61,13 @@ jobs:
           key:
             cache|${{ runner.os }}|${{ env.PY }}|${{ hashFiles('pyproject.toml') }}|${{
             hashFiles('poetry.lock') }}|${{ hashFiles('.pre-commit-config.yaml') }}
-      - run: python -m pip install poetry poetry-dynamic-versioning
+      - name: Install poetry
         shell: bash
-      - run: poetry install
+        run: |
+          python -m pip install poetry poetry-dynamic-versioning
+      - run: |
+          poetry run python -m pip install pip -U
```

Comment:
I still don't think we need this line. Did you add it for some specific reason?

Reply:
The following poetry install like will fail without it, at least for the python 3.6 environment. I guess it could be made to only apply in that part of the test matrix, though I don't know if it's worth the complexity.

This build demonstrates what I mean: https://github.com/copier-org/copier/runs/995092646?check_suite_focus=true

Reply:
But that message:

```
pip._vendor.pytoml.core.TomlError: /home/runner/work/copier/copier/pyproject.toml(75, 1): msg
```

Doesn't it mean there's a syntax error in the toml?

Reply:
well sort of; from the perspective of the pytoml implementation which doesn't properly support all of the toml spec. This seems to be the biggest problem with the PEP-517/pyproject.toml design; different libraries use different toml parsers with different limitations but they all have to parse the same file.

AFAIK the most compete toml parsing library available for python is tomlkit which poetry and poe both use.